qr-code/src/test/java/com/adyen/checkout/qrcode/internal/ui/DefaultQRCodeDelegateTest.kt

```diff
@@ -65,31 +76,91 @@ internal class DefaultQRCodeDelegateTest(
     private lateinit var statusRepository: TestStatusRepository
     private lateinit var paymentDataRepository: PaymentDataRepository
     private lateinit var delegate: DefaultQRCodeDelegate
-    private lateinit var fileDownloader: FileDownloader
+    private lateinit var imageSaver: ImageSaver
 
     @BeforeEach
     fun beforeEach() {
-        fileDownloader = spy(FileDownloader(context))
+        imageSaver = mock<ImageSaver>()
```

Comment:
I'd suggest to put this as `@Mock private val imageSaver: ImageSaver` in the constructor with the other mocks